src/app/song/song-tool-bar/song-tool-bar.component.ts

```diff
@@ -124,4 +132,18 @@ export class SongToolBarComponent implements OnInit {
     this.selectedPlaylists = [];
     }
   }
+
+  loadPermissionToConnectedUser() {
+    if(this.userService.connectedUser == undefined)
+    {
+        this.userService.getUserPermissionsUpdateListener().subscribe(user => {
+          this.hasOwnerPermissions = this.song.artists.some( 
+            artist => artist.username == user.username);
+        });
+    }
+    else {
+      this.hasOwnerPermissions = this.song.artists.some( 
+        artist => artist.username == this.userService.connectedUser.username);
+    }
+  }
```

Comment:
Is there an enforcement on unique username? Maybe we should compare using object id.